src/style.scss

```diff
@@ -4,89 +4,129 @@
+@import './node_modules/@alaskaairux/design-tokens/dist/tokens/SCSSVariables';
+@import './node_modules/@alaskaairux/design-tokens/dist/tokens/SassCustomProperties';
+@import './node_modules/@alaskaairux/design-tokens/dist/tokens/SCSSVariableMap';
 
-@import "./node_modules/@alaskaairux/webcorestylesheets/dist/breakpoints";
-@import "./node_modules/@alaskaairux/webcorestylesheets/dist/core";
-
-@import "./node_modules/@alaskaairux/webcorestylesheets/dist/utilityClasses/displayProperties";
+@import './node_modules/@alaskaairux/webcorestylesheets/dist/breakpoints';
+@import './node_modules/@alaskaairux/webcorestylesheets/dist/core';
 
+@import './node_modules/@alaskaairux/webcorestylesheets/dist/utilityClasses/displayProperties';
 $auro-inset-spacing-options: lg;
-$auro-inset-directions:'';
-@import "./node_modules/@alaskaairux/webcorestylesheets/dist/utilityMixins/insetUtility";
+$auro-inset-directions: '';
+@import './node_modules/@alaskaairux/webcorestylesheets/dist/utilityMixins/insetUtility';
 
 ::slotted(*:hover) {
   cursor: pointer;
 }
 
+:host(:not([data-show])) {
+  .popover {
+    @extend .util_displayHiddenVisually;
+  }
+}
+
+:host([data-show]) {
+  .popover {
+    z-index: var(--auro-depth-tooltip);
+
+    background-color: var(--auro-color-background-lightest);
+  }
+}
+
 :host([addSpace]) {
   .popover {
     margin: 1rem 0 !important;
   }
 }
 
 .popover {
-  border-radius: var(--auro-border-radius);
-  background: var(--auro-color-background-lightest);
-  box-shadow: -2px 0px 5px 2px rgba(0, 0, 0, 0.08),
-    0px 2px 5px 1px rgba(0, 0, 0, 0.08);
   display: inline-block;
+
   max-width: calc(100% - 2rem);
 
+  border-radius: var(--auro-border-radius);
+  background: var(--auro-color-background-lightest);
+  box-shadow: -2px 0 5px 2px rgba(0, 0, 0, .08),
+    0 2px 5px 1px rgba(0, 0, 0, .08);
+
   @include auro_breakpoint--sm {
     max-width: 50%;
   }
-
   @include auro_breakpoint--md {
     max-width: 40%;
   }
-
   @include auro_breakpoint--lg {
     max-width: 27rem;
   }
-
-  &:not([data-show]){
-    @extend .util_displayHiddenVisually;
-  }
-
-  &[data-show] {
-    z-index: var(--auro-depth-tooltip);
-    background-color: var(--auro-color-background-lightest);
-  }
 }
 
-.popover[data-popper-placement^="top"] > .arrow {
+.popover[data-popper-placement^='top'] > .arrow {
   bottom: -10px;
 
   &:before {
-    transform: rotate(45deg);
     top: -16px;
     left: -6px;
+
+    transform: rotate(45deg);
   }
 }
 
-.popover[data-popper-placement^="bottom"] > .arrow {
+.popover[data-popper-placement^='bottom'] > .arrow {
   top: -10px;
 
   &:before {
-    transform: rotate(-135deg);
     top: 4px;
     right: -6px;
+
+    transform: rotate(-135deg);
+  }
+}
+
+.popover[data-popper-placement^='top'] {
+  &:before {
+    position: absolute;
+    bottom: -20px;
+    left: 0;
+
+    display: block;
+
+    width: 100%;
+    height: 20px;
+
+    content: '';
+  }
+}
+
+.popover[data-popper-placement^='bottom'] {
```

Comment:
```suggestion
data-popper-placement^='bottom'] {
```

Reply:
Agreed about the best way to do it. I left it that way because that's how it was done in the previous/original implementation. There are multiple places this selector is so I'll do a patch that cleans it up in all locations.